descriptors/ModuleDescriptor-template.json

```diff
@@ -223,7 +223,7 @@
     },
     {
       "id": "patron-notice-policy-storage",
-      "version": "0.8",
+      "version": "0.9",
```

Comment:
Should an addition to an enumeration be considered a breaking change?
If we consider an `enum` to contain the entire universe of values, which is why it can be a constant in most languages, then adding `Request Expiration` to the `enum` would change the universe. Clients may not expect this new value, and, if treated like a Java enum, will not be able to map this value to an existing v0.8 value, likely causing an error. An `enum` isn't a `list` or `array` and I don't think we should treat it as such when considering backwards compatibility.

Reply:
It makes sense. Changed interface version to `2.0`.

Reply:
@mreno-EBSCO that’s a good question, one that likely deserves a considered response from me later.

This interface was pre 1.0, and therefore explicitly unstable, why choose now to not only mark it stable and to skip the first major version?

Reply:
@marcjohnson-kint  apologies, skiped the first major version by mistake.
It seems like `feeFineNotices` is not going to be used as there is different for notice configuration mechanism presented in mod-feefines. Probably it's better desicion to leave the version under 1.0. What do you think?

Reply:
@opochepam Yes, personally I would leave this interface experimental, especially given the feedback being generated as we apply further changes